### DISCUSSION TITLE
Unable to verify the Student Account

### DISCUSSION THREAD

**Discussion by sri113-collab** [community/community#143992] [GitHub Education]
### Hi everyone,

Hi I am trying to get GitHub student access and I have been rejected multiple times despite me uploading an official letter from the university confirming that I am enrolled in their MS Program via distance learning. I need help

**Comment by BDadmehr0** [comment]
**Ensure Your Letter Includes Key Details**

- **Your Full Name**: This should match the name on your GitHub account.
- **University Name and Official Seal/Stamp**: The institution’s official logo or stamp can help legitimize the document.
- **Current Enrollment Status**: Explicitly state that you are a currently enrolled student.
- **Program and Level of Study**: Mention that you are in an MS Program and clearly state “distance learning” or “remote” if relevant.
- **Enrollment Period/Expected Graduation**: If possible, add your enrollment start date or expected graduation date.

**Reply by sri113-collab** [reply]
I can upload the documents I have here but I am not sure what more is needed as the docs I included where my tuition bill copy for fall and a letter from the University confirming I am enrolled in the program for fall 2024, but it does not seem to help
